plugins/Sid/CMakeLists.txt

```diff
@@ -1,51 +1,15 @@
 INCLUDE(BuildPlugin)
 
-INCLUDE_DIRECTORIES(resid)
+if(NOT ${LMMS_HAVE_PERL})
+	message(STATUS "Cannot build Sid plugin, Perl not found")
```

Comment:
Could this go with the other similar messages in the plugin status list at the bottom of the root `CMakeLists.txt`? It would be more consistent and easier to find the message that way, whereas here it could easily get lost amongst unrelated output.